SmartTutorTests/DataFactories/SemanticProjectCheckerChallengeTestData.cs

```diff
@@ -0,0 +1,231 @@
+﻿namespace SmartTutor.Tests.DataFactories
+{
+    internal class SemanticProjectCheckerChallengeTestData
+    {
+        public static string[] GetFourPassingClasses()
+        {
+            return new[]
+           {
+@"
```

Comment:
This should be exported into a .txt file that is labeled to avoid compilation (we can discuss this today).